matlab/mex_fnft_nsep.c

```diff
@@ -52,17 +55,20 @@ void mexFunction(int nlhs, mxArray *plhs[], int nrhs, const mxArray *prhs[])
     if ( !mxIsDouble(prhs[1]) || mxGetM(prhs[1]) != 1 || mxGetN(prhs[1]) != 2 )
         mexErrMsgTxt("Second input T should be a double 1x2 vector.");
     if ( !mxIsDouble(prhs[2]) || mxGetNumberOfElements(prhs[2]) != 1 )
-        mexErrMsgTxt("Third input kappa should be a scalar.");
+        mexErrMsgTxt("Third input phase_shift should be a scalar.");
+    if ( !mxIsDouble(prhs[3]) || mxGetNumberOfElements(prhs[3]) != 1 )
+        mexErrMsgTxt("Fourth input kappa should be a scalar.");
 
     D = mxGetNumberOfElements(prhs[0]);
     K = D;
     M = D;
     T = mxGetPr(prhs[1]);
-    kappa = (int)mxGetScalar(prhs[2]);
+    phase_shift = mxGetScalar(prhs[2]);
```

Comment:
Could we make the `phase_shift` an optional parameter in the Matlab interface with default 0.0? (This version is per se fine, but it will break the Matlab examples in a paper that will be presented soon.)

Reply:
Yes that can be done. Changing it now.